doc/stdenv/stdenv.chapter.md

```diff
@@ -990,6 +990,22 @@ Convenience function for `makeWrapper` that replaces `<\executable\>` with a wra
 
 If you will apply it multiple times, it will overwrite the wrapper file and you will end up with double wrapping, which should be avoided.
 
+### `prependToVar` \<variableToPrependTo\> \<elements\> {#fun-prependToVar}
+
+Prepends elements to a variable. For example:
+
+```bash
+configureFlags="--disable-static"
+prependToVar configureFlags --disable-dependency-tracking \
+                            --enable-foo
+echo $configureFlags
+--disable-dependency-tracking --enable-foo --disable-static
+```
```

Comment:
It's not clear what's input and what's output here. As it stands, this is not a valid Bash script, as the last line will produce an error. I would recommend making the distinction semantically clear, like this:

````suggestion
```bash
configureFlags="--disable-static"
prependToVar configureFlags --disable-dependency-tracking --enable-foo
echo $configureFlags
```

    --disable-dependency-tracking --enable-foo --disable-static
````

But that approach is contested, and I don't think there is a clear resolution yet, see here: https://github.com/NixOS/nix/pull/7389

A possibly less controversial approach would be the following

````suggestion
```shellSession
$ configureFlags="--disable-static"
$ prependToVar configureFlags --disable-dependency-tracking --enable-foo
$ echo $configureFlags
--disable-dependency-tracking --enable-foo --disable-static
```
````

But that's really inconvenient to copy-paste for trying it out, because currently the manual rendering doesn't support automatically filtering out the shell prompt characters and the output lines from the selection.

Reply:
i much prefer the "less controversial approach" because using $ to denote a command is the standard in the unix world